package.json

```diff
@@ -69,6 +71,7 @@
     "eslint-plugin-ember": "^5.1.0",
     "loader.js": "^4.5.1",
     "normalize.css": "^7.0.0",
+    "nyc": "^11.6.0",
```

Comment:
do we need that only for the command line reporting? can't we use a built-in command line reporter?

Reply:
Problem here is that we use ember-exam to have parallel tests, so the process is as follows:

1. run ember tests with `COVERAGE=true` env-var (and `parallel: true` option in config/coverage.js)
2. do `ember coverage-merge` to merge the results from each parallel test run into a single './coverage' directory
3. display the code coverage results. here `nyc` is utilized. do you know how to display the results with the text reporter using only ember-cli-code-coverage/istanbul?